pipelines/validation/h1c_idr3_2/h1c_idr3_v2_validation.toml

```diff
@@ -78,7 +89,19 @@ actions = ["EXTRACT_AUTOS",
            # "CLEAN_FILES", # TODO: update this and put it back in
           ]
 
+[UNCALIBRATE]
+args = ["{basename}",
+        "${UNCALIBRATE_OPTS:sky_cmp}",
+        "${UNCALIBRATE_OPTS:path_to_sim_configs}",
+        "${UNCALIBRATE_OPTS:path_to_sim_files}",
+        "${UNCALIBRATE_OPTS:path_to_scripts}",
+        "${UNCALIBRATE_OPTS:path_to_data_files}",
+        "${UNCALIBRATE_OPTS:save_dir_base}",
+       ]
+
 [EXTRACT_AUTOS]
+prereqs = "UNCALIBRATE"
```

Comment:
You also need to make this a prereq for redcal.

Reply:
Gotcha. Will do.